libraries/x86/scripts/gpio.py

```diff
@@ -0,0 +1,16 @@
+import socket
```

Comment:
If the goal is to make a library for interacting with programs through external commands, wouldn't it be better to make a single client that can read/write with the program and send/receive arbitrary commands over creating a new connection for every single command?